client/landing/domains/registrant-verification/index.jsx

```diff
@@ -54,17 +54,20 @@ class RegistrantVerificationPage extends Component {
 	getVerificationSuccessState = ( domains ) => {
 		const { translate } = this.props;
 
-		const verifiedDomains = domains.join( ', ' );
+		const DomainLinks = domains.map( ( domain, index ) => [
+			index > 0 && ', ',
+			<a key={ domain } href={ `https://${ domain }?logmein=1` }>
+				{ domain }
+			</a>,
+		] );
 
 		return {
 			title: translate( 'Success!' ),
 			message: translate(
-				'Thank your for verifying your contact information for:{{br /}}{{strong}}%(domain)s{{/strong}}.',
+				'Thank your for verifying your contact information for:{{br /}}{{strong}}{{domainLinks /}}{{/strong}}.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.